editor/src/components/inspector/common/property-path-hooks.ts

```diff
@@ -768,60 +788,9 @@ export function useInspectorLayoutInfo<P extends LayoutProp | StyleLayoutProp>(
     untransformValue,
     createLayoutPropertyPath,
   )
-  if (
-    !inspectorInfo.propertyStatus.set &&
-    !inspectorInfo.propertyStatus.controlled &&
-    specialSizeMeasurements != null
-  ) {
-    const measuredValue = getValueFromSpecialSizeMeasurements(property, specialSizeMeasurements)
-    if (measuredValue != null) {
-      inspectorInfo.value = measuredValue
-      inspectorInfo.propertyStatus.detected = true
-      inspectorInfo.controlStatus = getControlStatusFromPropertyStatus(inspectorInfo.propertyStatus)
-    }
-  }
   return inspectorInfo
 }
 
-function getValueFromSpecialSizeMeasurements<P extends LayoutProp | StyleLayoutProp>(
-  property: P,
-  specialSizeMeasurements: SpecialSizeMeasurements,
-): ParsedProperties[P] | null {
-  // TODO: are there other properties in special size measurementes to extract?
-  let value: number | undefined = undefined
-  switch (property) {
-    case 'paddingLeft':
-      value = specialSizeMeasurements.padding.left
```

Comment:
do we use specialSizeMeasurements.padding and specialSizeMeasurements.margin anywhere else? if not, we could remove them. but maybe some canvas controls use them?

Reply:
We have `margin-controls.tsx` and `padding-controls.tsx` that use them